W/WhiteboxTools/build_tarballs.jl

```diff
@@ -0,0 +1,46 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg, BinaryBuilderBase
+
+name = "WhiteboxTools"
+version = v"2.0.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/jblindsay/whitebox-tools/archive/refs/tags/v$(version).tar.gz", "18705fc948bdb2f96cd816e5a72d36b9cc460aa8c910383d23fdbd61641aab60")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/whitebox-tools-*/
+
+cargo build --release
+mkdir /workspace/destdir/bin/
+find target/${rust_target}/release/ -maxdepth 1 -type f -executable | xargs -I '{}' mv {} "${bindir}"
+install_license LICENSE.txt
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms(; experimental=true)
+filter!(p -> !Sys.iswindows(p) || arch(p) != "i686", platforms)
+filter!(p -> libc(p) != "musl" || proc_family(p) != "arm", platforms)
+
```

Comment:
Did you check that these platforms don't work? I excluded them from librsvg because there was something specific to it in the generated code that wasn't compatible with those platforms, but in general the Rust toolchain should work there.